frontend/src/components/style/Historic.scss

```diff
@@ -0,0 +1,40 @@
+$gradientColor1: #00aac6;
+$gradientColor2: #00c1b7;
+$gradientColor3: #00dc9f;
+$gradientColor4: #ffffff;
+$gradientColor5: #72bac52a;
+$gradientColor6: #00abc62a;
+
+.HistoricBox {
+  display: flex;
+  .ScrollBar {
```

Comment:
why is it empty ? normal ?

Reply:
the scrollbar will be stylized in the future